README.md

```diff
@@ -85,6 +85,8 @@ cpu, err := ghw.CPU(ghw.WithChroot("/host"))
 
 You can make `ghw` read from snapshots (created with `ghw-snapshot`) using
 environment variables or programmatically.
+Please check `SNAPSHOT.md` to learn more about how GHW creates and consumes
```

Comment:
nit: here and throughout, please s/GHW/\`ghw\`/ :)

Reply:
fixed, thanks!